tests/features/custom_pages/location_finder.feature

```diff
@@ -2,56 +2,89 @@
 Feature: Location finder
   I want to make sure that Location Finder Page shows new branches. Given some content exists.
 
+  Background: Create necessary content for tests
+    Given I create branch content:
+      | title                |
+      | Behat Branch 01 YMCA |
+      | Behat Branch 02 YMCA |
+      | Behat Branch 03 YMCA |
+      | Behat Branch 04 YMCA |
+    And I create large camp content:
```

Comment:
Any reason why Camp should have all fields. While Branches are having just titles?
Notes for future improvements: 
1) Add all fields to Branch and check that all necessary field appear on /locations page.
2) May be we don't need to create both Branches and Camps in the Background. Because not all tests require that.